src/CheckoutSdk/Consts.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Checkout.Sdk
+{
+    public struct Consts
```

Comment:
I don't see the benefit of centralising these constants. The idea was to encapsulate the knowledge of the source type within the relevant class. Since 2/3 of these constants are only used in a single place, I do not believe this is necessary.

Reply:
My assumption was that if one of them has more than one reference, then quite possibly the rest will have that too. This was done to avoid the idea of 'magic strings' as well as to be able to easily find occurrences of concept that seem fairly important to the design. Generally this could be avoided with using enums, but as the strings have already been chosen as identifiers, this would require the developer to do additional work to parse enum name (therefore encouraging to use magic strings again) so constants seemed like a better idea. Do you think it harms the solution somehow?